lucille-core/pom.xml

```diff
@@ -17,11 +17,15 @@
   <name>Lucille Core</name>
 
   <dependencies>
-
     <dependency>
       <groupId>org.opensearch.client</groupId>
-      <artifactId>opensearch-rest-high-level-client</artifactId>
-      <version>${opensearch-rest-high-level-client.version}</version>
+      <artifactId>opensearch-rest-client</artifactId>
+      <version>2.9.0</version>
```

Comment:
lucille-core/pom.xml has a mix of hardcoded versions with variables defined in lucille-parent/pom.xml
If we're hardcoding 2.9.0 here and not using ${opensearch-rest-high-level-client.version} we should remove ${opensearch-rest-high-level-client.version} from lucille-parent/pom.xml
But before we do that, perhaps @kyle-kmw could comment on when versions should be specified in lucille-core/pom.xml versus in lucille-parent/pom.xml

Reply:
Done